generation/new_client_hermetic_build/README.md

```diff
@@ -0,0 +1,160 @@
+image
+([gcr.io/cloud-devrel-public-resources/java-library-generation](https://pantheon.corp.google.com/gcr/images/cloud-devrel-public-resources/global/java-library-generation))
+to generate a new library. It also creates the pull request.
+
+
+## Execute the Github Action
+
+You in order to run the
+[Github Action](https://github.com/googleapis/google-cloud-java/actions/workflows/generate_new_client_hermetic_build.yaml)
+, you need to specify a few parameters.
+These parameters will be available in the Cloud Drop link (a YAML file) included in the buganizer request.
+The example in this README uses AlloyDB's [Cloud Drop](https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml) file as an example.
+
+### API short name (`api_shortname`)
+
+For convenience of the subsequent commands, define a variable for API short name.
+This value will be used by default to generate the following:
+* `--distribution-name`
+* `--destination-name`
+
+The corresponding value in the Cloud Drop page is `api_short_name`.
+
+Example: `alloydb`
+
+> [!IMPORTANT]
+> `api_short_name` is not always unique across client libraries.
+> In the instance that the `api_short_name` is already in use by an existing client library, you will need to determine a unique name.
+> See example under [Advanced Options](#Example with duplicate api_short_name).
+
+### Proto path (`proto_path`)
+
+The script takes "proto path" parameter. This is the path from the internal `google3/third_party/googleapis/stable` root to the
+directory that contains versions (e.g., "v1" or "v2").
+Note that the internal `google3/third_party/googleapis/stable` directory is mirrored externally in https://github.com/googleapis/googleapis/blob/master/.
+
+For example, if the buganizer ticket includes:
+
+> Link to protos: `http://...(omit).../google/cloud/alloydb/v1alpha/alloydb_v1alpha.yaml`.
+
+then the corresponding external mirrored proto is here: https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1alpha/alloydb_v1alpha.yaml.
+
+Therefore, the "proto path" value we supply to the command is `google/cloud/alloydb`.
+
+We will publish a single module for a service that includes all versions in this path. Once the service has been published once, any future additional versions will automatically be generated via OwlBot.
+
+### Name pretty (`name_pretty`)
+
+The corresponding value in the Cloud Drop page is `title`.
+
+Example: `AlloyDB API`
+
+### Product Docs (`product_docs`)
+
+The corresponding value in the Cloud Drop page is `documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs`
+
+### REST Docs (`rest_docs`)
+
+The corresponding value in the Cloud Drop page is `rest_reference_documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
```

Comment:
Do you mean add it to the action parameter?

Reply:
This is an advanced option, I modified the doc accordingly.
Changed to
```
If the value exists, add it as a flag to the python command below (see [Advanced
Options](#advanced-options]):
`--rpc-docs="https://cloud.google.com/speech-to-text/docs/reference/rpc" \`
```